sklearn/linear_model/bayes.py

```diff
@@ -550,12 +550,34 @@ def fit(self, X, y):
         self.scores_ = list()
         coef_old_ = None
 
+        def choose_pinvh_singular_value_cutoff_threshold(sigma_inverse_):
+            """
+            Determine a suitable 'cond' parameter value to scipy.linalg.pinvh.
+
+            Since Scipy 1.3.0 the default value is smaller and reduces
+            ARDRegression stability ( Github issue #14055)
+
+            Currently just reproduces Scipy behaviour prior to 1.3.0.
+            """
+            dtype_code = sigma_inverse_.dtype.char.lower()
+            factor = {'f': 1E3, 'd': 1E6}
+            pinvh_cond_factor = factor[dtype_code] * np.finfo(dtype_code).eps
+            s, u = linalg.eigh(sigma_inverse_, lower=True, check_finite=False)
```

Comment:
`eigh` is going to be called again in `pinvh`, we might just as well port the whole `pinvh`

Reply:
Sounds sensible, good way to pin behaviour across Scipy versions. Where should such 'vendored' code go in the scikit-learn codebase - 'utils.pinvh', or maybe 'utils.linalg'?

Reply:
If it's a lot of code, you can put it under `sklearn/externals/_scipy_linalg.py` then import it or not depending on the version from `sklearn/utils/fixes.py`.

If it's not that much code, you can put in in `utils.fixes` directly.